src/prune-fhir-schema.ts

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+//import fhir_schema from '../schema/fhir-schema-full.json';
+import fs from 'fs';
+import JSZip from 'jszip'
+import got from 'got'
+
+
+const fullSchemaLink = 'https://hl7.org/fhir/fhir.schema.json.zip'
```

Comment:
`https://hl7.org/fhir/R4/fhir.schema.json.zip` is a better long-term stable link

Reply:
Thanks.  I updated the link.